packages/amplify-util-uibuilder/types/amplify-codegen.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module 'amplify-codegen' {
```

Comment:
Why is this needed? Can this type be exported from the `amplify-codegen` package instead?

Reply:
I asked the team owning the package and was told they do not plan to publish types. Hand-writing my own types was required due to our plugin having `"strict": true` in our tsconfig while other plugins referencing the library do not.